cmd/vfsgendev/vfsgendev.go

```diff
@@ -0,0 +1,188 @@
+// vfsgendev is a convenience tool for using vfsgen in a common development configuration.
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/ast"
+	"go/build"
+	"go/parser"
+	"go/token"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"text/template"
+)
+
+var (
+	sourceFlag = flag.String("source", "", "Specifies the http.FileSystem variable to use as source.")
+	nFlag      = flag.Bool("n", false, "Print the generated source but do not run it.")
+)
+
+const (
+	sourceTags = "dev"
+
+	outputFilename = "data_generate.go"
+	outputTags     = "!dev"
+)
+
+type source struct {
+	ImportPath   string
+	PackageName  string
+	VariableName string
+}
+
+var errInvalidFormat = fmt.Errorf("invalid format")
```

Comment:
nit: I prefer `errors.New` for things that don't need formatting -- just a silly suggestion :)